### PR TITLE
DEM-935 Implement results page navigation

### PR DESCRIPTION
**Pull request by yordanalexandrov**
+ Added page navigation component wrapping MatPaginator
+ Added tests

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -1,32 +1,62 @@
...
     "no-unused-vars": "off",
-    "@typescript-eslint/no-unused-vars": "error",
     "no-array-constructor": "off",
-    "@typescript-eslint/no-array-constructor": ["error"],
     "valid-jsdoc": "off",
     "comma-dangle": "off",
     "no-invalid-this": "off",
-    "@typescript-eslint/explicit-function-return-type": "error"
-  }
+    // disable TypeScript rule for all files
```
Comment (svilenvelikov) [inline .eslintrc.json]: hmmm don't these comments break the json?!

**Review by svilenvelikov — APPROVED** [review]

**Review by svilenvelikov — CHANGES_REQUESTED** [review]
The build has failed

File: .eslintrc.json (hunk 1 above)
Comment (yordanalexandrov) [reply .eslintrc.json]: Hm. You might be right. I saw it like this elsewhere. I will remove them.

**Comment by yordanalexandrov** [comment]
> The build has failed

Fixed it

**Review by svilenvelikov — APPROVED** [review]

**Review by teodossidossev — APPROVED** [review]

**Pull request merged**